setup.py

```diff
@@ -12,7 +12,7 @@
           'backoff==1.8.0',
           'requests==2.23.0',
           'singer-python==5.9.0',
-          'twitter-ads==9.0.1'
+          'twitter-ads==10.0.0'
```

Comment:
Version 9 is totally deprecated, so to pass the build we have updated the SDK version.